pkg/nsx/services/nsxserviceaccount/cluster.go

```diff
@@ -0,0 +1,289 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package nsxserviceaccount
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/bindings"
+	mpmodel "github.com/vmware/vsphere-automation-sdk-go/services/nsxt-mp/nsx/model"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
+	"github.com/vmware-tanzu/nsx-operator/pkg/logger"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+	"github.com/vmware-tanzu/nsx-operator/pkg/util"
+)
+
+const (
+	siteId             = "default"
+	enforcementpointId = "default"
+	PortRestAPI        = "rest-api"
```

Comment:
Unused constant 'PortRestAPI'

Reply:
This is for external use and will be used by operator in future.